addons/libs/images.lua

```diff
@@ -407,6 +426,9 @@ windower.register_event('mouse', function(type, x, y, delta, blocked)
             if meta[dragged.image].root_settings then
                 config.save(meta[dragged.image].root_settings)
             end
+            if x == clicked.x and y == clicked.y then
+                call_events(dragged.image, 'left_click')
+            end
             dragged = nil
```

Comment:
Like `dragged` before, `clicked` should be unset here.

Reply:
Thanks, done!